src/components/AdminContainer.js

```diff
@@ -148,20 +150,19 @@ class AdminContainer extends Component {
     return (
       <React.Fragment>
         {adminAddress ? (
-          <Panel>
-            {baseStyle}
-            <div className="flex">
-              <div className="w-50">
-                <h1 className="mt0">Admin</h1>
-              </div>
-              <div className="w-50">
-                <NetworkSelectorContainer />
-              </div>
-            </div>
-            <div className="flex bb pb3">
-              <div className="w-50">
-                <h3>
-                  Current account{" "}
+          <React.Fragment>
+            <nav className="dt pa3 w-100 border-box pa3 ph5-ns bg-white shadow-1-ns flex-l flex-row-l">
```

Comment:
Can we not span it across the entire page but add a container here? Looks odd on a big screen

Reply:
Updated.